controllers/buildpipeline/buildpipeline_adapter.go

```diff
@@ -64,12 +65,12 @@ func NewAdapter(pipelineRun *tektonv1beta1.PipelineRun, component *applicationap
 // to the build PipelineRun being processed exists. Otherwise, it will create a new pipeline Snapshot.
 func (a *Adapter) EnsureSnapshotExists() (controller.OperationResult, error) {
 	if !h.HasPipelineRunSucceeded(a.pipelineRun) {
+		a.logger.Info("Build pipeline has not succeeded")
```

Comment:
Do we need this log here? It may be repeated multiple times, not sure if it gives any value.

Reply:
No, that was something I added for debugging that I forgot to remove.